### PR TITLE
Corrected nearly all the issues found by SonarLint

### PR DESCRIPTION
**Pull request by trapprakam**

### CODE REVIEW

File: contribs/src/main/java/com/netflix/conductor/contribs/kafka/KafkaEventQueueProvider.java [hunk 1]
```diff
@@ -22,43 +22,46 @@
 public class KafkaEventQueueProvider implements EventQueueProvider {
     private static final Logger logger = LoggerFactory.getLogger(KafkaEventQueueProvider.class);
     protected Map<String, KafkaObservableQueue> queues = new ConcurrentHashMap<>();
```
Comment (weispa) [inline contribs/src/main/java/com/netflix/conductor/contribs/kafka/KafkaEventQueueProvider.java]: can queues be final ?

File: server/src/main/java/com/netflix/conductor/bootstrap/Main.java [hunk 2]
```diff
@@ -33,25 +33,26 @@
...
+        final Optional<EmbeddedElasticSearch> embeddedElasticSearch = serverInjector.getInstance(EmbeddedElasticSearchProvider.class).get();
         embeddedElasticSearch.ifPresent(BootstrapUtil::startEmbeddedElasticsearchServer);
 
         BootstrapUtil.setupIndex(serverInjector.getInstance(IndexDAO.class));
 
         try {
             serverInjector.getInstance(IndexDAO.class).setup();
-        } catch (Exception e) {
+        } catch (final Exception e) {
             e.printStackTrace(System.err);
```
Comment (weispa) [inline server/src/main/java/com/netflix/conductor/bootstrap/Main.java:55]: should use proper logging here, and not print stack trace

File: server/src/main/java/com/netflix/conductor/bootstrap/Main.java [hunk 3]
```diff
@@ -65,14 +66,14 @@ public static void main(String[] args) throws Exception {
         System.out.println(" \\___\\___/|_| |_|\\__,_|\\__,_|\\___|\\__\\___/|_|   ");
```
Comment (weispa) [inline server/src/main/java/com/netflix/conductor/bootstrap/Main.java:65]: use a logger *@Slf4j for example

File: contribs/src/main/java/com/netflix/conductor/contribs/kafka/resource/handlers/ResourcesLoader.java [hunk 4]
```diff
@@ -66,7 +64,7 @@ public void locateResources() {
             while (urls.hasMoreElements()) {
                 readJarFile(urls.nextElement());
             }
-        } catch (IOException ex) {
+        } catch (final IOException ex) {
             logger.error("Can not locate Resources for Kafka Listener", ex);
```
Comment (weispa) [inline contribs/src/main/java/com/netflix/conductor/contribs/kafka/resource/handlers/ResourcesLoader.java]: be consistent with error logging, ie pass the ex.getMessage() as a param to the log as with other logging

File: contribs/src/main/java/com/netflix/conductor/contribs/kafka/KafkaEventQueueProvider.java (hunk 1 above)
Comment (trapprakam) [reply contribs/src/main/java/com/netflix/conductor/contribs/kafka/KafkaEventQueueProvider.java]: Queues are not final, they only have a protected access

File: server/src/main/java/com/netflix/conductor/bootstrap/Main.java (hunk 2 above)
Comment (trapprakam) [reply server/src/main/java/com/netflix/conductor/bootstrap/Main.java:55]: I didn't write the code in main class. That was there from Netflix

File: server/src/main/java/com/netflix/conductor/bootstrap/Main.java (hunk 3 above)
Comment (trapprakam) [reply server/src/main/java/com/netflix/conductor/bootstrap/Main.java:65]: Same

File: contribs/src/main/java/com/netflix/conductor/contribs/kafka/resource/handlers/ResourcesLoader.java (hunk 4 above)
Comment (trapprakam) [reply contribs/src/main/java/com/netflix/conductor/contribs/kafka/resource/handlers/ResourcesLoader.java]: Corrected

File: contribs/src/main/java/com/netflix/conductor/contribs/kafka/KafkaEventQueueProvider.java (hunk 1 above)
Comment (weispa) [reply contribs/src/main/java/com/netflix/conductor/contribs/kafka/KafkaEventQueueProvider.java]: but they're only initialised once right, regardless of whether they're protected. if so, they can be final unless you're re initialising them in any super class which is not a good idea anyway.

File: server/src/main/java/com/netflix/conductor/bootstrap/Main.java (hunk 2 above)
Comment (weispa) [reply server/src/main/java/com/netflix/conductor/bootstrap/Main.java:55]: bad netflix

**Review by weispa — APPROVED** [review]

File: contribs/src/main/java/com/netflix/conductor/contribs/kafka/KafkaEventQueueProvider.java (hunk 1 above)
Comment (trapprakam) [reply contribs/src/main/java/com/netflix/conductor/contribs/kafka/KafkaEventQueueProvider.java]: Corrected

**Pull request merged**